packages/flutter/test/material/dialog_test.dart

```diff
@@ -2895,11 +2895,84 @@ void main() {
     expect(simpleDialog.insetPadding, isNull);
   });
 
+  // This is a regression test for https://github.com/flutter/flutter/issues/153983.
```

Comment:
Did you just remember this?

Reply:
Added comment for a test added in https://github.com/flutter/flutter/pull/154173 when resolving git conflicts.